editor/src/components/editor/npm-dependency/npm-dependency.ts

```diff
@@ -333,6 +333,20 @@ export function dependenciesWithEditorRequirements(
   return [...userDefinedDeps, ...EditorTypePackageDependencies]
 }
 
+export function orderedDependenciesEqual(
+  l: Array<RequestedNpmDependency>,
+  r: Array<RequestedNpmDependency>,
+): boolean {
+  return (
+    l.length === r.length &&
+    l.reduce(
+      (matches: boolean, next: RequestedNpmDependency, index: number) =>
+        matches && shallowEqual(next, r[index]),
```

Comment:
Why `shallowEqual`? Does that run the risk of a change to `RequestedNpmDependency` breaking unexpectedly?

Reply:
Hmmm tbh we could probably just use `fastDeepEquals` and bin off this function

Reply:
Fixed in a647bdbb